templates.go

```diff
@@ -633,37 +633,28 @@ write_files:
         prometheus.io/port: '10254'
         prometheus.io/scrape: 'true'
     spec:
-      replicas: 3
+      replicas: {{len .Params.Cluster.Workers}}
```

Comment:
`{{.Params.Cluster.Workers | len}} is also possible. I don't know which is better.